src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,7 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    int widthOfScoop;
```

Comment:
The field 'widthOfScoop' should be private. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters).